README.md

```diff
@@ -18,26 +18,6 @@ Via Composer
 $ composer require lahaxearnaud/laravel-u2f
 ```
 
-### Provider
```

Comment:
Perhaps you should say that the package is "auto discoverable" to avoid any confusions.

Reply:
Sure, done ;-)